### PR TITLE
[Compute] vm create: Add warning when specifying accelerated networking and an existing NIC together

### PR DESCRIPTION
**Pull request by qwordy**
Issue #9495 

Accelerated networking is an option of NIC. When using an existing NIC, should not specifying accelerated networking. Add warning first, change to error 6 months later - April 2020.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

File: src/azure-cli/HISTORY.rst [hunk 1]
```diff
@@ -38,6 +38,7 @@ Release History
 * vm create: Use standard public IP SKU automatically when using zones.
 * vm create: Compose a valid computer name from VM name if computer name is not provided.
 * vm create: Add --workspace to enable log analytics workspace automatically.
+* vm create: Do not allow specifying accelerated networking when using an existing NIC.
```
Comment (MyronFanQiu) [inline src/azure-cli/HISTORY.rst]: Well. It must be a breaking change. Right? Should we deprecated it first?

Comment (qwordy) [reply src/azure-cli/HISTORY.rst]: Kind of. Can't use --accelerated-networking and --nics togother now. But no one is deprecated. What's the workflow of this condition?

Comment (qwordy) [reply src/azure-cli/HISTORY.rst]: I think I can use warning first. Then after several releases, change it to error.

Comment (qwordy) [reply src/azure-cli/HISTORY.rst]: Resolved. Added a reminder to onenote to change it to error later.

**Pull request merged**